molecule/default/converge.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Converge
+  hosts: all
+  become: true
+
```

Comment:
Just to reiterate what we talked about this morning - 

Vars (similar to what's demonstrated in the readme) go here, and then in verify.yml, that's where you check to prove the role did what it was supposed to.

If you find the need to duplicate data, you can avoid that making a separate vars file, and including it both here and in `verify.yml` with `vars_files: ./shared-vars.yml` .

Reply:
Modified, Please review.